armi/reactor/assemblies.py

```diff
@@ -129,12 +129,16 @@ def __lt__(self, other):
         """
         Compare two assemblies by location.
 
-        This allow consistent sorting of assemblies based on location. If either
```

Comment:
probably should all be in a `Notes` thing talking about implementation

Reply:
Good point. Should be better now.